recipes/cpr/all/conandata.yml

```diff
@@ -28,10 +28,8 @@ patches:
       base_path: "source_subfolder"
     - patch_file: "patches/003-1.4.0-curl-global-scope.patch"
       base_path: "source_subfolder"
-  "1.5.0":
-    - patch_file: "patches/001-1.5.0-curl-global-scope.patch"
-      base_path: "source_subfolder"
-    - patch_file: "patches/002-1.5.0-not_search_for_openssl.patch"
+  "1.5.2":
+    - patch_file: "patches/005-fix-curl-components.patch"
```

Comment:
Now the entry for `1.5.2` is duplicated... only one of those patches will be applie

Reply:
Thankful it looks like the right one was. 

Fixed 🤞